binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/handler/HueBridgeHandler.java

```diff
@@ -241,7 +242,9 @@ public void initialize() {
             }
             onUpdate();
         } else {
-            logger.warn("Cannot connect to hue bridge. IP address or user name not set.");
+            String warn = "Cannot connect to hue bridge. IP address or user name not set.";
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.OFFLINE.CONFIGURATION_ERROR, warn);
+            logger.warn(warn);
```

Comment:
Question to all: Now that we have the message in the status, do we still consider it a recommendation to also log the message (and then even as a warning)? Feels a bit redundant now. If log information about thing status changes is desired, this could actually be taken directly from the events (as it is done for the bus.events log file for item events already). Any opinions?